editor/import/editor_scene_importer_gltf.h

```diff
@@ -31,6 +31,7 @@
 #ifndef EDITOR_SCENE_IMPORTER_GLTF_H
 #define EDITOR_SCENE_IMPORTER_GLTF_H
 
+#include "core/core_string_names.h"
```

Comment:
This is used in the `.cpp`, so it should be included in the `.cpp`.

Reply:
👍 fixed